src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_TryCatch.cs

```diff
@@ -3115,6 +3115,233 @@ void F(int p)
             VerifyFlowGraphForTest<BlockSyntax>(compilation, expectedGraph);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void TryFlow_26()
```

Comment:
>TryFlow_26 [](start = 20, length = 10)

Could you please add a test like this but with a returned expression?

Reply:
> Could you please add a test like this but with a returned expression?

There is one already - ReturnFlow_13, it is in the other test file

---
In reply to: [171912558](https://github.com/dotnet/roslyn/pull/25190#discussion_r171912558) [](ancestors = 171912558)